src/api/patron-codes.md

```diff
@@ -125,4 +125,10 @@ Request][polling] using the transacting APIs.
 }
 ```
 
+{% h4 Error Responses %}
+
+| Status |          Code                 |          Description              |
+| :----- | :-----------------------------| :---------------------------------|
+| 403    | {% break _ CHECKSUM_FAILED %} | Luhn checksum digit doesn't pass. |
```

Comment:
actually, should these be 400 bad request?

Reply:
Gone with 400 for checksum, and 410 Gone for expired.